pkg/testutil/testutil.go

```diff
@@ -19,80 +19,48 @@ import (
 	"strings"
 	"time"
 
-	"github.com/pingcap/check"
 	"github.com/pingcap/kvproto/pkg/pdpb"
 	"github.com/stretchr/testify/require"
 	"google.golang.org/grpc"
 )
 
 const (
-	defaultWaitRetryTimes = 200
-	defaultSleepInterval  = time.Millisecond * 100
-	defaultWaitFor        = time.Second * 20
+	defaultWaitFor      = time.Second * 20
```

Comment:
Why repeat it?  The similar code may be found in `/client/testutil.go`.

Reply:
This is to prevent the client module from relying on the PD module.